_posts/2016-12-08-underhanded-rust.en.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: post
+title:  "The 2016 Underhanded Rust Contest"
+date:   2016-12-07 11:23:34 -1000
+categories: underhanded
+lang: en-US
+---
+
+# Introduction
+
+We’re happy to announce the first annual Underhanded Rust Contest, a
+competition that tests our assertion that Rust, and our ecosystem, is safe.
```

Comment:
So, some of this stuff will be about safety, but some of it won't. It's tough since "safe" is very specific in Rust; I would imagine the _best_ underhanded entries should be 100% safe code.